src/main/java/com/flansmod/client/model/RenderGun.java

```diff
@@ -455,94 +391,153 @@ public void renderGun(ItemStack item, GunType type, float f, ModelGun model, Gun
 
 		// Get all the attachments that we may need to render
 		AttachmentType scopeAttachment = type.getScope(item);
-		AttachmentType barrelAttachment = type.getBarrel(item);
-		AttachmentType stockAttachment = type.getStock(item);
 		AttachmentType gripAttachment = type.getGrip(item);
-		AttachmentType gadgetAttachment = type.getGadget(item);
-		AttachmentType slideAttachment = type.getSlide(item);
-		AttachmentType pumpAttachment = type.getPump(item);
-		AttachmentType accessoryAttachment = type.getAccessory(item);
-
-		ItemStack scopeItemStack = type.getScopeItemStack(item);
-		ItemStack barrelItemStack = type.getBarrelItemStack(item);
-		ItemStack stockItemStack = type.getStockItemStack(item);
-		ItemStack gripItemStack = type.getGripItemStack(item);
-		ItemStack gadgetItemStack = type.getGadgetItemStack(item);
-		ItemStack slideItemStack = type.getSlideItemStack(item);
-		ItemStack pumpItemStack = type.getPumpItemStack(item);
-		ItemStack accessoryItemStack = type.getAccessoryItemStack(item);
-
-		// Gun recoil
 		animations.recoilAmount = model.recoilAmount;
 
-		GL11.glPushMatrix();
```

Comment:
hm, where's this been moved to?

Reply:
Actually, thats prob not accurate numbers cause my code has further changes, but its good

Reply:
Oke